L1Trigger/L1TGEM/python/me0TriggerPseudoDigis_cff.py

```diff
@@ -37,3 +37,16 @@
     me0Segments,
     me0TriggerConvertedPseudoDigis
 )
+
+from RecoLocalMuon.GEMRecHit.gemRecHits_cfi import *
+from RecoLocalMuon.GEMSegment.gemSegments_cfi import *
+
+ge0TriggerPseudoDigiTask = cms.Task(
+    simMuonME0PseudoReDigisCoarse,
```

Comment:
I don't think we should be using the ME0 Pseudo digis for GE0, the lines below the comment should be all thats needed for GE0, unless I misremember the setup?

Reply:
@watson-ij  I believe you're right; I'll remove them and run a test to see if this causes any issues.